Client/Frontend/Browser/URLBarView.swift

```diff
@@ -467,9 +462,8 @@ class URLBarView: UIView {
             self.rightBarConstraint?.updateOffset(URLBarViewUX.URLBarCurveOffset + URLBarViewUX.URLBarCurveBounceBuffer + tabsButton.frame.width)
 
             // Make the editable text field span the entire URL bar, covering the lock and reader icons.
-            self.locationTextField.snp_remakeConstraints { make in
+            self.locationTextField.snp_updateConstraints { make in
```

Comment:
`snp_updateConstraints` is used to update the constants of constraints. Shouldn't this be `snp_remakeConstraints`?